### ISSUE TITLE
Offset + sticky element = problem

### ISSUE THREAD

**Issue by korvinko** [homerjam/angular-sticky-container#1]
Hi. I use your library and it is good tool. But I found problem with firefox.  If I define offset and element have position sticky it create gap between container  and element but other browser work corrects. Could you please remove offset for firefox browsers?

Firefox:
![image](https://cloud.githubusercontent.com/assets/2064465/8763877/9db776be-2dbb-11e5-9f9f-91349652fbfc.png)

Сhrome and IE:
![image](https://cloud.githubusercontent.com/assets/2064465/8763878/a7cc55e8-2dbb-11e5-827f-81bdcab8ded8.png)

I resolve problem by change top property to zero :

``` php
  if (customModernizr.csspositionsticky && options.enableSticky) {
                        targets = angular.element($element[0].querySelectorAll(options.targetSelector));

                        angular.forEach(targets, function(target) {
                            var hjStickyScrollOffset = parseInt(angular.element(target).attr('hj-sticky-scroll-offset')) || 0;

                            target = angular.element(target);

                            target.css('position', 'sticky');
                            target.css('position', prefix.css + 'sticky');
                            target.css('top', 0 + 'px');
                        });
                    }
```

**Comment by homerjam** [comment]
Hi,

I've checked the demo in the latest version of Firefox and everything seems fine.

Can you create a demo showing the problem so that I can look into it some more please?

Thanks

James

**Comment by korvinko** [comment]
Hi, I upload my site here : http://burovsego.ru/
But here I already change JS file as described upper:

``` javascript
target.css('top', 0 + 'px'); // for Firefox
```

Sticky block '.l-top-content-outer'. If it not enough I will prepare another demo.

**Comment by homerjam** [comment]
Yes, please create a simplified demo with just the minimum elements to reproduce the error. Thanks
